packages/graphql-transformers-e2e-tests/src/__tests__/ConnectionsWithAuthTests.e2e.test.ts

```diff
@@ -276,7 +276,7 @@ test('Testing reading an owner protected field as a non owner', async () => {
     console.log(response1);
     expect(response1.data.createFieldProtected.id).toEqual("1")
     expect(response1.data.createFieldProtected.owner).toEqual(USERNAME1)
-    expect(response1.data.createFieldProtected.ownerOnly).toEqual("owner-protected")
+    expect(response1.data.createFieldProtected.ownerOnly).toBeNull()
```

Comment:
I don't understand why this test should be updated. The field is protected to only allow the owner, but USERNAME1 is creating the object and USERNAME1 is set as the owner field. ownerOnly should not be null for the owner. Possible source of issue left as comment above in the protectPrivateFields check block.

Reply:
@yuth You may want to review these test changes as you have marked this PR as approved.

Reply:
The behavior of per field auth has changed - the PR for docs elaborates on this more
[Docs PR #868 L941](https://github.com/aws-amplify/docs/pull/868/files#diff-8076c2db248c895d2ade586451fa9b55R944#L941)